patches/chrome-installer-util-BUILD.gn.patch

```diff
@@ -10,18 +10,15 @@ index 7782a268825828b437943047cd3cc98b70537d53..979d53fb5595abdcdda955186ba5d851
  }
  
  # Use this version of installer_util to link to the generated strings in .rc
-@@ -241,7 +242,18 @@ generate_embedded_i18n("generate_strings") {
- 
+@@ -242,6 +243,16 @@ generate_embedded_i18n("generate_strings") {
    output_file_name_base = "installer_util_strings"
  
--  branding = branding_path_component
+   branding = branding_path_product
 +  brand = "$branding_path_component"
```

Comment:
This patch is a mess. We should be setting branding_path_component in config.js, not here. We should also remove the comments and do a follow-up ticket because the brave_strings.grd thing is wrong as well, but requires more work to fix. Another patch is causing this to be a problem
```
 grit("chromium_strings") {
-  source = "chromium_strings.grd"
+  source = "${branding_path_component}_strings.grd"
```
we should just be using chromium_strings.grd instead of patching branding_path_component and this

Reply:
Simplified slightly in [32de785](https://github.com/brave/brave-core/pull/15294/commits/32de78500e975e82b295ed319687d07a1503dff5). I left the comment in for now so that the info isn't lost for the follow up. There's more involved with brave_strings.grd than just these 2 patches.

Reply:
Based on DM, additional fix via https://github.com/brave/brave-core/pull/15294/commits/43759de1477baab7fd8dc59d90f1f2f468783c8d